test/test_suite.py

```diff
@@ -328,7 +329,17 @@ class DateTest(DateFixtureTest, _DateTest):
     and maintain DRY concept just inherit the class to run tests successfully.
     """
 
-    pass
+    @pytest.mark.skipif(
+        bool(os.environ.get("SPANNER_EMULATOR_HOST")), reason="Skipped on emulator"
```

Comment:
nit: You don't need to explicitly cast this to a bool; Python will do it automatically.

Reply:
First I've tried to write the code without `bool` and it failed with `SyntaxError`. It was three commits ago:
https://github.com/cloudspannerecosystem/python-spanner-sqlalchemy/pull/94/commits/9da642c37599847a88e908e4bc9267230b9dd1a0